scylla-cql/src/frame/request/batch.rs

```diff
@@ -42,7 +42,7 @@ pub enum BatchStatement<'a> {
 
 impl<'a, StatementsIter, Values> Request for Batch<'a, StatementsIter, Values>
 where
-    StatementsIter: Iterator<Item = BatchStatement<'a>> + Clone,
+    StatementsIter: ExactSizeIterator<Item = BatchStatement<'a>> + Clone,
```

Comment:
This diff looks like an unnecessary leftover. I should probably revert it while addressing review comments, if any.